Exercises-03/exercise-01.js

```diff
@@ -0,0 +1,17 @@
+function flattenArray(arr) {
+    let flattenedArray = [];
+    if(!Array.isArray(arr)) return 0
+    for(let element of arr){
+        if(Array.isArray(element)){
+            const nestedArr = flattenArray(element);
```

Comment:
I would suggest using the full names in variable naming ex: `nestedArr` -> `nestedArry`